File: MarketComponents/BackEnd/src/main/java/com/backApi/demo/Dto/AuthResponse.java
package com.backApi.demo.Dto; // Asegúrate de que el paquete sea correcto, puedes crear 'Dto' si no existe

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuthResponse {
    private String message;
    private String token; // Opcional: para futuros tokens JWT
    private Integer userId; // Opcional: para devolver el ID del usuario logeado
    private String email; // Opcional: para devolver el email del usuario logeado
}

File: MarketComponents/MiTIendaApp/app/src/main/java/com/example/myapplication/model/AuthResponse.kt
package com.example.myapplication.model // Puedes ponerlo en .model o crear .data.response

import com.google.gson.annotations.SerializedName

data class AuthResponse(
    val message: String,
    val token: String? = null,
    @SerializedName("userId") // Asegúrate de que el nombre del campo JSON coincida
    val userId: Int? = null,
    val email: String? = null
)